packages/react-instantsearch/scripts/build.sh

```diff
@@ -2,15 +2,32 @@
 
 set -e # exit when error
 
+# copy files
 mkdir -p dist/ &&
 rm -rf dist/* &&
 cp package.json dist/ &&
-cp README.md dist/ &&
+cp README.md dist/
+
+NODE_ENV=production
+
+# first make es module build
+export BABEL_ENV=es
+mkdir -p dist/es &&
```

Comment:
no, the published directory on npm is `dist` alone, see https://unpkg.com/react-instantsearch@beta/